rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -3021,8 +3024,7 @@ void main_window::dropEvent(QDropEvent* event)
 			return;
 		}
 
-		Emu.SetForceBoot(true);
-		Emu.Stop();
+		Emu.GracefulShutdown(false);
```

Comment:
you should also add a GracefulShutdowns after the first check in AddGamesFromDir.
(not before its usage, since the user still has a chance to abort closing a running game by canceling the file dialog)